Package.swift

```diff
@@ -21,10 +20,6 @@ let package = Package(
       name: "PactConsumerSwift",
       dependencies: ["BrightFutures", "Nimble", "Result"],
       path: "./Sources"
-    ),
-    .testTarget(
```

Comment:
Removing this fails the tests stage in the `build.sh` script as `swift test` command doesn't know where to look for tests.

Reply:
Yeah, so, we build and test with carthage. I don't think its necessary to run the tests with spm as well....

Feels like we are making things difficult by using two build systems. The Package.swift should just exist for downstream users who want to pull in this library using SPM (similar to how we support Cocoapods).